templates/default/delos.css

```diff
@@ -11287,20 +11288,30 @@ p.il_calevent .btn-link {
 }
 td.calevent {
   vertical-align: top;
-  padding: 3px;
+  /*padding: 3px;*/
   border-spacing: 0px;
   border-collapse: collapse;
   border-width: 1px;
   border-style: solid;
   border-color: #ddd;
   font-size: 90%;
-  background-color: white;
+  /*background-color:white;*/
+  height: 1px;
+  position: relative;
 }
 div.calevent {
   font-size: 90%;
   padding: 3px;
   margin: 3px;
 }
+div.calfullcontent {
+  position: absolute;
```

Comment:
Using absolute positioning is not an option here, since it prevents such a div to be proberly wrapped (e.g. by some wrapper dropzone). Please look for an alternative. Checkout: https://stackoverflow.com/questions/6319500/how-to-make-the-wrapping-div-element-with-relative-position-match-child-elements?rq=1.

Reply:
The solution proposed in stackoverflow doesn't work for us, we need it the other way around. It only works if we have the "inner" height adapting the "wrap" height and its exactly what we don't have. I've also tried with height:100% in different ways but firefox and chorme work differently.  One option could be this small js code at the end of the week template:
    <script>
    $('.calfullcontent').each(function(){
    var td_height = parseInt($(this).parent().height());
    $(this).css('height',td_height);
    });
    </script>

What do you think?

Reply:
ugly. but since I can not come up with something bettter atm, I guess you can use this as fallback.

Reply:
Yes I know, we will change it as soon as we found a better solution.